.gitignore

```diff
@@ -19,3 +19,7 @@ coverage
 
 # bundled files
 lib/bundle.js
+
+package-lock.json
```

Comment:
@shouston3 it's super _noisy_ and does not get used by Travis-CI https://github.com/travis-ci/travis-ci/issues/7936